src/sail_sv_backend/jib_sv.ml

```diff
@@ -592,17 +604,22 @@ module Make (Config : CONFIG) = struct
     in
     Str.string_match regexp s 0
 
-  let pp_id_string id =
+  let is_sv_global s =
+    match !sv_globals with
+    | None -> failwith "USED is_sv_global BUT sv_globals IS UNSET"
```

Comment:
This should use the Sail error reporting mechanism `raise (Reporting.err_general l <msg>)` so it prints a message formatted in the standard way with location information.

Reply:
(Although without the global variable this function shouldn't exist)